[camera][iOS] Reject from takePictureAsync if camera is not ready (#7144)
expo/expo: docs/pages/versions/unversioned/sdk/camera.md

### `takePictureAsync()`

Takes a picture and saves it to app's cache directory. Photos are rotated to match device's orientation (if **options.skipProcessing** flag is not enabled) and scaled to match the preview. Anyway on Android it is essential to set `ratio` prop to get a picture with correct dimensions.

> > **Note** be sure not to call this method before [`onCameraReady`](./#oncameraready) callback is fired. 

Reviewer: ```suggestion
> **Note**: Make sure to wait for the [`onCameraReady`](./#oncameraready) callback before calling this method.
```

just a small nitpicky change
